crates/pathfinder/src/rpc/api.rs

```diff
@@ -378,8 +391,47 @@ impl RpcApi {
         request: Call,
         block_hash: BlockHashOrTag,
     ) -> RpcResult<Vec<CallResultValue>> {
-        let call = self.sequencer.call(request.into(), block_hash).await?;
-        Ok(call.result)
+        use futures::future::TryFutureExt;
+        let seq = self
+            .sequencer
+            .call(request.clone().into(), block_hash)
+            .map_ok(|x| x.result)
+            .map_err(Error::from);
+
+        match self.call_handle.as_ref() {
+            Some(h) => {
+                let local = h.call(request, block_hash).map_err(Error::from);
+                let (local, seq) = tokio::join!(local, seq);
+
+                match (local, seq) {
+                    (Ok(x), Ok(y)) if x == y => {
+                        println!("got equal to sequencer response: {x:?}");
```

Comment:
Whats the plan with the `println!`s? Will these become `logs` at some point?

Reply:
Naah, those will be there for when we can test this, and then hopefully remove them. I am actually currently dragging a very few non-pushed commits along in my wip branches, so thought to put it in right away.

This is only activated if the python stuff worked in the start (and wasn't like, commented out), and it doesn't affect any of the tests either.

Reply:
Yes but this will confuse the user? So its useful for us to see when trying things out?

Reply:
Yes, I guess it would confuse the user if it was left in after we get this working. My point is that until we can test this feature this uglyness should be in for the time being so that it's available for everyone instead of me maintaining this as part of a patch set along with other printlns which are our only hope of logging at the moment.

I think the situation is similar to including the forged block_hash in #94 (root hash mixed with block number) as a crutch to get the L1 sync moving forwards was better than to have zero working sync in the tree.